app/questionnaire/router.py

```diff
@@ -146,7 +140,7 @@ def get_first_incomplete_location_in_survey(self):
             if location:
                 return location
 
-        return self.get_last_section_last_block_location()
+        return self.get_last_location_in_survey()
```

Comment:
We should change `get_first_incomplete_location_in_survey` to `get_first_incomplete_location_in_questionnaire` for consistency.

Reply:
@MebinAbraham You were the one that suggested get_last_location_in_survey

Reply:
Yes because it is consistent with the other method. My point is, if we want to use `questionnaire` instead, then need to make sure we change the other reference also.

Reply:
Sticking with survey, Mebin is right we need to be consistent, and if I start pulling the questionnaire/survey thread I am going to be here all night, sounds a nice easy refactoring piece if we do want to change all of them across the app